proposals/blittable.md

```diff
@@ -139,22 +102,26 @@ The primary drawback of this feature is that it serves a small number of develop
 
 Yet these frameworks are are often the basis for the majority of .NET applications out there.  Hence performance / correctness wins at this level can have a riple effect on the .NET ecosystem.  This makes the feature worth considering even with the limited audience. 
 
-There will also likely be a small transition period after this is released where core libraries move to adopt it.  Types like [System.Runtime.InteropServices.ComTypes.FILETIME](https://msdn.microsoft.com/en-us/library/system.runtime.interopservices.comtypes.filetime(v=vs.110).aspx) are common in interop code.  Until it is marked as `blittable` in source though, developers won't be able to depend on it in their libraries.  
-
 ## Alternatives
 [alternatives]: #alternatives
 
 There are a couple of alternatives to consider:
 
 - The status quo:  The feature is not justified on its own merits and developers continue to use the implicit opt in behavior.
-- Generic constraint only: The blittable keyword is used on generic constraints only.  This allows for developers to author efficient helper libraries.  But the types involved lack any declarative support and hence are fragile across distributed development. 
 
+## Resolved questions
+[resolved]: #resolved-questions
+
+### Blittable vs. Unmanaged
+The F# language has a very [similar feature](https://docs.microsoft.com/en-us/dotnet/articles/fsharp/language-reference/generics/constraints) which uses the keyword unmanaged. The blittable name comes from the use in Midori.  May want to look to precedence here and use unmanaged instead. 
+
+**Resolution** The language decide to use unmanaged 
+
+### Verifier
 
-## Unresolved questions
-[unresolved]: #unresolved-questions
+Does the verifier / runtime need to be updated to understad the use of pointers to generic type parameters?  Or can it simply work as is without changes. 
```

Comment:
Requiring that pointer's underlying type is unmanaged is a limitation of C#.  I.E.  `string*` is a valid type at CLR level.
Verification treats any pointer type as unverifiable.

I think we should be ok here without changes. That said, there could be bugs.

Reply:
Ah that's a misunderstanding on my part then. I will fix.